test/externalTests/sign.js

```diff
@@ -22,19 +22,13 @@ module.exports = () => async (bot) => {
 
         assert.strictEqual(sign.signText, '1\n2\n3\n')
 
-        if (sign.blockEntity && sign.blockEntity.Text1) {
-          assert.strictEqual(sign.blockEntity.Text1.toString(), '1')
-          assert.strictEqual(sign.blockEntity.Text2.toString(), '2')
-          assert.strictEqual(sign.blockEntity.Text3.toString(), '3')
-          assert.strictEqual(sign.blockEntity.Text4.toString(), '')
-        }
-
         if (sign.blockEntity) {
           // Check block update
           bot.activateBlock(sign)
           assert.notStrictEqual(sign.blockEntity, undefined)
         }
 
+        bot._client.removeAllListeners('open_sign_entity')
```

Comment:
Why? This seems like it could conflict with mineflayer's internal event handlers.

Reply:
It was causing issues when the world was unloaded, for some reason `open_sign_entity` was being triggered if I remember correctly, so the next tests `beforeAll` was failing, seems like some 1.20 thing. I checked when I did it and Mineflayer doesn't use `open_sign_entity` internally.

Reply:
This should be using once then, I've updated the PR